### PR TITLE
use RTC for round-tripping video from device A, through pipeline on device B, and back to A 

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: pkg/Library/NodeGraph/NodeInspector.js [hunk 1]
```diff
@@ -161,9 +161,9 @@ async constructConnectedStore(connection, selected, pipeline, nodeTypes, service
   const value = await Promise.all(selected?.map(
     async ({from, store}) => {
       const node = pipeline.nodes.find(node => node.key == from);
-      const nodeType = this.findNodeType(node?.name, nodeTypes);
-      if (nodeType) {
-        return await this.getBindingValue(store, nodeType.$stores[store], node, service);
+      if (node) {
```
Comment (mariakleiner) [inline pkg/Library/NodeGraph/NodeInspector.js:164]: could you, please, revert this change? this is a PR i made earlier this week and it didn't merge well, i think

**Review by mariakleiner — APPROVED** [review]

File: pkg/Library/NodeGraph/NodeInspector.js (hunk 1 above)
Comment (sjmiles) [reply pkg/Library/NodeGraph/NodeInspector.js:164]: better?

**Review by mariakleiner — APPROVED** [review]

**Review by jinjingforever — APPROVED** [review]
